SshAgentLib/PageantAgent.cs

```diff
@@ -286,6 +287,39 @@ public void StopMsysSocket()
       msysSocket = null;
     }
 
+    /// <summary>
+    /// Starts a wsl style socket that can be used by the ssh program
+    /// that comes with wsl.
+    /// </summary>
+    /// <param name="path">The path to the socket file that will be created.</param>
+    public void StartWslSocket(string path)
+    {
+      if (disposed) {
+        throw new ObjectDisposedException("PagentAgent");
+      }
+      if (wslSocket != null) {
```

Comment:
Should we throw an exception here instead of silently failing?

Reply:
I copied this over from `StartMsysSocket` and `StartCygwinSocket` though. Might be worth changing along with those if you do that at some point?